### DISCUSSION TITLE
how setup while we use `mono` workspace of vscode

### DISCUSSION THREAD

**Discussion by tclxtyc15129** [0no-co/GraphQLSP#247] [Q&A] [answered]
### Describe the bug

it seems that it can not generated `graphql-env.d.ts` file while we have config in tsconfig.json
```json
 "compilerOptions": {
    "plugins": [
      {
        "name": "@0no-co/graphqlsp",
        "schema": "http://localhost:5001/admin-api",
        "tadaOutputLocation": "./src/graphql-env.d.ts"
      }
    ],
```
<img width="375" alt="image" src="https://github.com/0no-co/gql.tada/assets/144304047/e8267586-0a3d-4365-b7a7-643db57b0539">

### gql.tada version

latest version

### Validations

- [X] I can confirm that this is a bug report, and not a feature request, RFC, question, or discussion, for which [GitHub Discussions](https://github.com/0no-co/gql.tada/discussions) should be used
- [X] Read the [docs](https://gql-tada.0no.co/).
- [X] Follow our [Code of Conduct](https://github.com/0no-co/gql.tada/blob/main/CODE_OF_CONDUCT.md)

**Comment by JoviDeCroock** [comment, marked as answer]
It's the same as for any other project, relative to the tsconfig location you will need to point to your schema and ensure that you're not using the VSCode TS version but the workspace one. You can open the tsserver logs to debug your specific issue. Moving this issue over to the appropriate repo though

**Comment by tianyingchun** [comment]
yes, it works, 

<img width="428" alt="image" src="https://github.com/0no-co/GraphQLSP/assets/2833251/89cd5f2f-5712-4183-b052-22ce4a94d8f8">


--- root
--- app.code-workspace

   ```json
     {
      "folders": [
        {
          "name": "app",
          "path": "packages/app",
        },
        {
          "name": "root",
          "path": ".",
        },
      ],
     "settings": {        
      "typescript.tsdk": "root/node_modules/typescript/lib",
      "typescript.enablePromptUseWorkspaceTsdk": true
      }
     }
  ```
--- app
---  tsconfig.json

   ```json
   "compilerOptions": {
   ....
    "plugins": [
      {
        "name": "@0no-co/graphqlsp",
        "schema": "http://localhost:3001/admin-api",
        "tadaOutputLocation": "./src/apollo/graphql-env.d.ts"
      }
    ]
}
```
